spec/lib/va_profile/contact_information/service_spec.rb

```diff
@@ -82,6 +82,16 @@
         end
       end
     end
+
+    context 'when person response has no body data' do
```

Comment:
The bug occurs when the response has no bio section, but the code calling `Person.build_from` is very cautious and could also occur if there is no body, so I've written the test for the broader case. 
```
      def self.from(raw_response = nil)
        @response_body = raw_response&.body

        new(
          raw_response&.status,
          person: VAProfile::Models::Person.build_from(@response_body&.dig('bio'))
        )
      end
```